src/pages/BorrowerProfile/fundedBorrowerProfile.vue

```diff
@@ -1,23 +1,30 @@
 <template>
 	<www-page>
-		<div class='row'>
-			<!-- Borrower photo -->
-			<!-- Borrower Name -->
-			<p> {{ loan.name }} </p>
-			<!-- Total funded/loan amount -->
-			<p> {{ loan.use }}</p>
-			<p> {{ loan.description }}</p>
-			<!-- Funded State/ FUNDED! -->
-			<p> {{ loan.status }}</p>
-			<!-- Borrower location -->
-			<p> {{ loan.geocode.city }}</p>
-		</div>
-		<div class='row'>
-			<!-- Loan use statement -->
-		</div>
-		<div class='row'>
-			<!-- Borrower story -->
+		<!-- Borrower photo -->
+		<div class="row">
+			<div class="small-6 columns">
+				<loan-card-image
+					:loan-id="loan.id"
+					:name="loan.name"
+					:retina-image-url="loan.image.retina"
+					:standard-image-url="loan.image.default"
+					v-kv-track-event="['basket', 'basket-loan-profile', 'basket-loan-profile']"
+					:open-in-new-tab="true"
```

Comment:
There's a prop flag for this you may want to pass the "useDefaultStyles"/"use-default-styles" set to false so you don't inherit the styles meant for the original loan card.